server/cmd/mmctl/commands/import.go

```diff
@@ -240,10 +250,28 @@ func importUploadCmdF(c client.Client, command *cobra.Command, args []string) er
 func importProcessCmdF(c client.Client, command *cobra.Command, args []string) error {
 	importFile := args[0]
 
+	isLocal, _ := command.Flags().GetBool("local")
+	isLegacy, _ := command.Flags().GetBool("legacy")
+	// in local mode, we tell the server to directly read from this file.
+	if isLocal && !isLegacy {
+		if _, err := os.Stat(importFile); errors.Is(err, os.ErrNotExist) {
+			return fmt.Errorf("file %s doesn't exist. NOTE: If this file was uploaded to the server, please use the --legacy flag to revert to old behavior.", importFile)
+		}
```

Comment:
We've introduced a new flag `isLegacy`, and changed the default behavior of not providing this value. Does this require a new major version of the tool?

Reply:
I've been debating this a bit. I am starting to think you are right. I'll revert this and go the opposite route. Thanks Michael.